recipes-nemomobile/mce/mce/10mce.ini

```diff
@@ -2,10 +2,10 @@
 
 [Modules]
 
-# Path to modules
+# Path to modules, usually architecture dependant
```

Comment:
architecture? you mean watch ? or soc ?

Reply:
Honest answer, I don't now. This is what upstream did so I just copied that part too 
https://github.com/AsteroidOS/mce/pull/9/files#diff-3c3643a28be8621cbc0d0738040f0050a32a4bb983481e7344cbb4526d23369eR5 and https://git.sailfishos.org/mer-core/mce/blob/master/inifiles/mce.ini#L5